app/services/calculators/subject_matter_of_dispute_disregard_calculator.rb

```diff
@@ -11,7 +11,6 @@ def initialize(capital_summary:, maximum_disregard:)
 
     def value
       total_disputed_asset_value = disputed_capital_value +
-        disputed_property_value +
```

Comment:
This calculator could perhaps be renamed to indicate that it's calculating _some_ of the SMOD disregard. My fear is that if it turns out the Correct order we're told to apply is `everything else -> main dwelling -> additional properties` then it will be very important that PropertyCalculator is called after "SubjectMatterOfDisputeDIsregardCalculator" in CapitalCollator, but it won't be at all clear from the names _why_.

(Also @exonian, just to flag up that counter to what we said at stand-up this morning, depending on the Correct order of applying SMOD, this code will be somewhere between trivially easy and slightly more complex to adapt. E.g. if the correct order has vehicles before main home and savings after, we'll need to break up SubjectMatterOfDisputeDisregardCalculator into two separate chunks etc)